components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -293,7 +249,11 @@ open class GleanInternalAPI {
 
         when (pingEvent) {
             Glean.PingEvent.Background -> {
-                sendPing("baseline", "baseline")
+                if (baselinePingScheduler.canSendPing()) {
+                    sendPing(BaselinePingScheduler.STORE_NAME, BaselinePingScheduler.STORE_NAME)
+                    baselinePingScheduler.updateSentTimestamp()
```

Comment:
We don't really do anything in `canSendPing` and `updateSentTimestamp`, so no need to have more code... right? :)